Sources/Extension/Alert+BSImagePicker.swift

```diff
@@ -0,0 +1,112 @@
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+import UIKit
+import Photos
+
+extension AssetsViewController {
+    internal func showAlertForRestricedOrNotDeterminedAccess(){
+        
+        let localizedTitle = NSLocalizedString("bsimagepicker.restrictedAccess.alert.title",
+                                               value: "Allow access to your photos",
+                                               comment: "Title of alert controller")
+        
+        let localizedMessage = NSLocalizedString("bsimagepicker.restrictedAccess.alert.message",
+                                                 value: "This lets you share from your camera roll and enables other features for photos. Go to your settings and tap \"Photos\".",
+                                                 comment: "Message of alert controller")
+        
+        let notNowLocalizedText = NSLocalizedString("bsimagepicker.restrictedAccess.alert.secondaryButton.title",
+                                                    value: "Not now",
+                                                    comment: "Secondary button title")
+        
+        let openSettinsgLocalizedText = NSLocalizedString("bsimagepicker.restrictedAccess.alert.openSettings.title",
+                                                          value: "Open Settings",
+                                                          comment: "Primart button title")
+        
+        let alert = UIAlertController(title: localizedTitle, message: localizedMessage, preferredStyle: .alert)
+        
+        let notNowAction = UIAlertAction(title: notNowLocalizedText, style: .cancel, handler: nil)
+        alert.addAction(notNowAction)
+        
+        let openSettingsAction = UIAlertAction(title: openSettinsgLocalizedText, style: .default) { [unowned self] (_) in
+            // Open app privacy settings
+            gotoAppPrivacySettings()
+        }
+        alert.addAction(openSettingsAction)
+        present(alert, animated: true, completion: nil)
```

Comment:
I get crash on iPad because the popoverPresentationController?.sourceView is not set. Send here the "manage" button so alert/sheet is displayed properly